### PR TITLE
login, signup and password page added

### PR DESCRIPTION
**Pull request by vaibhav18matere**
lps

### CODE REVIEW

File: auth/login.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        </ul>
+        <div class="auth-container flex-center">
+            <div class="auth-main-container flex-center box-shadow">
+                <div class="auth-title">
+                    <h2 class="text-center">Login</h2>
+                </div>
+                <div class="auth-main fc-centre">
+                    <div class="auth-email f-column">
+                        <label for="mail">Enter your e-mail</label>
+                        <input placeholder="user123@gmail.com" class='text-input' type="text">
```
Comment (RakshithBhat03) [inline auth/login.html:36]: You can use `type="email"` if you want.

**Review by RakshithBhat03 — APPROVED** [review]
Everything looks good. The code is clean and well structured.

**Review by TusharJade — APPROVED** [review]
Code is clean, Good to go👍

**Review by Radheshyam1234 — COMMENTED** [review]
looks good to merge

**Review by manojasarsa — APPROVED** [review]
You can use utility classes for flex properties. Rest it looks good!

**Pull request merged**